app/views/helpers/FullWidthContent.scala.html

```diff
@@ -0,0 +1,25 @@
+@*
```

Comment:
You don't need to add a `FullWidthContent` helper as one is already provided by `play-frontend-hmrc`. It's better practice to use the predefined helpers where possible because if they update the markup we inherit it with a simple version bump.

You can delete this view and import `uk.gov.hmrc.govukfrontend.views.html.components.FullWidthPageLayout` instead.

See:
https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/FullWidthPageLayout.scala.html